plugins/generator-1.18.1/forge-1.18.1/procedures/entity_set_offhand_item.java.ftl

```diff
@@ -0,0 +1,7 @@
+<#include "mcitems.ftl">
+if (${input$entity} instanceof LivingEntity _entity) {
+	ItemStack _setstack = ${mappedMCItemToItemStackCode(input$item, 1)};
+	_setstack.setCount(${opt.toInt(input$amount)});
+	_entity.setItemInHand(InteractionHand.OFF_HAND, _setstack);
+	if (_entity instanceof ServerPlayer _serverPlayer) _serverPlayer.getInventory().setChanged();

```

Comment:
Check here differs from check in mainhand item procedure you ported

Reply:
:heavy_check_mark: Fixed